src/Sylius/Component/Order/Model/Order.php

```diff
@@ -435,7 +435,9 @@ public function getAdjustmentsTotal($type = null)
 
         $total = 0;
         foreach ($this->getAdjustments($type) as $adjustment) {
-            $total += $adjustment->getAmount();
+            if (!$adjustment->isNeutral()) {
+                $total += $adjustment->getAmount();
+            }
```

Comment:
Yes it's definitely a domain decision and one that I understand. But if you're making assumptions already then I think you either make them fully, or don't make them at all.

It starts to make me wonder if shipping adjustments should have been placed on the `Shipment` instead....